data/src/main/java/com/globant/data/mapper/PokemonSupertypeRoomMapper.kt

```diff
@@ -0,0 +1,21 @@
+package com.globant.data.mapper
+
+import com.globant.data.database.entity.PokemonSupertypeRoom
+import com.globant.domain.entity.PokemonSupertype
+
+class PokemonSupertypeRoomMapper : BaseMapper<List<PokemonSupertypeRoom>, MutableList<PokemonSupertype>, MutableMap<String, Int>?> {
```

Comment:
this should be PokemonSupertypeDataBaseMapper

Reply:
refactored, thanks